doc/design_AsyncNDRegisterAcessor_and_NumericAddressedBackend.dox

```diff
@@ -0,0 +1,60 @@
+namespace ChimeraTK {
+/**
+\page design_AsyncNDRegisterAccessor_NumericAddressed Design: AsyncNDRegisterAccessor and the asynchronous implelentation in the NumericAddressedBackend
+
+\section  design_AsyncNDRegisterAccessor The AsyncNDRegisterAccessor
+
+The AsyncNDRegisterAccessor is an extension of the NDRegisterAccessor for push-type data.
+
+The NDRegisterAccessor itself extends the TransferElement with a user-type buffer. The TransferElement does not have a user type buffer and serves as a base class such that implementations with different user types can be stored together in containers. The TransferElement also contais a notification queue to implement the asynchronous read functionality. This notification queue is of type void (notifications only, no data is transported) because the data type is not known. The AsyncNDRegisterAccessor now povides the data transport queue which is needed for asynchronous accessors with data. As this queue is comparatively resource-intensive (multiple instances of the user buffer object pre-allocated in each queue) and not needed for synchronous data transfer, it is not directly added in the NDRegisterAccessor but in a second class.
+
+The data transport queue is implmented as continuation of the notification queue, such that the read operations of the notification queue trigger the filling of the user buffer.
```

Comment:
implmented -> implemented

Reply:
This is the wrong direction. The notification queue is a continuation of the data transport queue.